styles.css

```diff
@@ -523,59 +506,399 @@ form li + li {
+    height: 252px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+    width: 1850px;
+    margin-top: 1600px;
+    background-image: url(images/aboutmeimage.png);
+    background-size: cover;
+    background-repeat: repeat-x;
+  }
+
+  .after-resume {
+    display: flex;
+    flex-direction: row;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    gap: 30px;
+    width: 100%;
+  }
+
+  .aboutme {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    width: 400px;
+    height: 189px;
+    margin-bottom: 80px;
+  }
+
+  .titleaboutme {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 900px;
+    font-size: 40px;
+    line-height: 40px;
+    color: #172b4d;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .aboutmeparagraph {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    width: 770px;
+    height: 109px;
+    text-align: center;
+    color: #42526e;
+  }
+
+  .projectbuttonresume {
+    align-items: center;
+    font-family: 'Inter', sans-serif;
+    letter-spacing: 0.05em;
+    padding: 12px;
+    gap: 10px;
+    width: 200px;
+    height: 50px;
+    bottom: 15px;
+    font-weight: 550px;
+    font-size: 20px;
+    position: relative;
+    border: none;
+    background: #36b37f;
+    border-radius: 4px;
+    color: #fff;
+    cursor: pointer;
+  }
+
+  .languages,
+  .frameworks,
+  .skills {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+    background: #ebf0ee;
+    border: 1px solid #c1c7d0;
+    border-radius: 8px;
+    gap: 20px;
+    width: 349px;
+    height: 352px;
+    justify-content: center;
+  }
+
+  .contactandform {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .contactinfo {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    height: 400px;
+    width: 1185px;
+    margin-top: 60px;
+  }
+
+  .sendamessage {
+    width: 100%;
+    margin-top: 80px;
+    display: flex;
+    justify-content: left;
+  }
+
+  .contactparagraph {
+    width: 1500px;
+    height: 256px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 52px;
+  }
+
+  #name,
+  #email,
+  #message {
+    width: 700px;
+  }
+
+  .my-email,
+  #enddash {
+    display: none;
+  }
+
+  .get-in-button {
+    font-family: 'Inter', sans-serif;
+    letter-spacing: 0.05em;
+    padding: 12px;
+    gap: 10px;
+    width: 180px;
+    height: 50px;
+    font-weight: 550px;
+    font-size: 20px;
+    border: none;
+    background: #36b37f;
+    border-radius: 4px;
+    color: #fff;
+    cursor: pointer;
+    margin-bottom: 20px;
+    margin-top: 10px;
+    float: left;
+  }
+
+  .about-social-container {
```

Comment:
:orange_circle: [OPTIONAL] Kindly make sure the style and alignment of these icons follow the Figma Template

|Figma Design| Your Design|
|:----:|:-----:|
|![Screenshot from 2023-01-26 23-16-08](https://user-images.githubusercontent.com/26736582/214956083-df751e01-9cad-4ee3-ac32-49ac4702ba3a.png)|![Screenshot from 2023-01-26 23-37-17](https://user-images.githubusercontent.com/26736582/214956209-06e7ed49-93a4-47a0-995d-f14f99cc31b9.png)|

Please make sure you add the missing border and update the font-color

Reply:
@bienvenuushindi do you referr to the space below the social icons? Or you referr to a rectangle that is part of the above design. 
![rectangle](https://user-images.githubusercontent.com/120115282/214962998-ea4d0033-8954-49c8-8871-400c0fb9bbae.jpg)